silk/tests/test_silk.py

```diff
@@ -0,0 +1,47 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import mock
+import pytest
+
+from datadog_checks.silk import SilkCheck
+
+from .common import BLOCKSIZE_METRICS, METRICS, READ_WRITE_METRICS
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+def test_check(aggregator, instance, dd_run_check):
+    check = SilkCheck('silk', {}, [instance])
+    dd_run_check(check)
+
+    for metric in METRICS:
+        aggregator.assert_metric(metric)
```

Comment:
Can we also assert that the expected silk tags are getting collected?

Reply:
I wasn't able to get expected tags to be asserted since each volume of the Silk instance will result in a different `volume_name` tag. There are some shared tags between all Silk metrics, although I am not sure how to assert for those.

Reply:
You can use `assert_metric_has_tag` to assert individual tags. I think you can also store a list of the `volume_name`s and then iterate through them while asserting metrics with those tags. Here's an example: https://github.com/DataDog/integrations-core/blob/master/snmp/tests/test_profiles.py#L184)

Reply:
I got `assert_metric_has_tag` to cover the base tags, although I think getting all of the metric-specific tags might get a bit messy since there are so many varied tags that are dependent on the metric itself with only some overlap between metrics.